lib/Doctrine/Common/Cache/Psr6/DoctrineProvider.php

```diff
@@ -122,4 +123,13 @@ protected function doGetStats()
     {
         return null;
     }
+
+    public function reset()
```

Comment:
```suggestion
    public function reset(): void
```

Let's move this new public method above the protected ones.